source/scss/_pf-reboot.scss

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2016 Red Hat, Inc
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//
+// Typography
+// --------------------------------------------------
+ol,ul,dl{
+  margin-bottom:0;
+}
+p{
+  margin-bottom: 0;
```

Comment:
I would not remove this globally, sometimes we need to use paragraph inside a component, like in cards.